### PR TITLE
Refresh border coloring in graph view without having to refresh the entire page

### PR DESCRIPTION
**Pull request by KevYuen**
**Work in Progress**: Creating PR to get feedback on feature

Currently, the only way to get an updated graph is to refresh the entire page. 
This PR adds a button that on the UI that remove the need to refresh the entire page. 

**Questions:**
Should we have the UI automatically poll the server for update rather then a manual click?
If not, what is the best place to place the button?
Which other view should we expand this feature to first?(ex. gantt, tree, etc...)

**Use Case:**
1. Open dag graph view
2. Run a task
3. Click refresh button and the border colors will be updated to fail/running/succeeded without having to refresh the entire page

screen shot 2016-01-31 at 4 14 57 pm

### CODE REVIEW

**Comment by mistercrunch** [comment]
Nice! Let's start with the button for now, it would be easy to switch it to poll with a timer. Design-wise you're probably fine with the refresh icon only (no text). 

Ideally there's not both a GO and and a refresh button.

**Comment by KevYuen** [comment]
Changed the location of the button to be more in context of the graph itself:
screen shot 2016-02-06 at 4 11 55 pm

Hopefully this is nicer.

**Comment by mistercrunch** [comment]
This is great, I'm hoping I get some time to test and merge this week!

File: airflow/www/templates/airflow/graph.html [hunk 1]
```diff
@@ -268,5 +274,17 @@
         return false
     }
 
+    d3.select("#refresh_button").on("click",
+        function() {
+            $.get(
```
Comment (mistercrunch) [inline airflow/www/templates/airflow/graph.html:294]: We need some sort of indicator that things are getting fetched here, maybe the loading.gif count elsewhere.

Also error handling the .get call. What if the async call fails?

Comment (KevYuen) [inline airflow/www/templates/airflow/graph.html:294]: Will add those when I have sometime this week.

**Comment by KevYuen** [comment]
Added a banner for errors and loading gif when it's fetching the data.

**Comment by mistercrunch** [comment]
This is great. Thanks for the feature!

**Pull request merged**